src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMParams.scala

```diff
@@ -17,6 +17,15 @@ trait LightGBMExecutionParams extends Wrappable {
   def getParallelism: String = $(parallelism)
   def setParallelism(value: String): this.type = set(parallelism, value)
 
+  val topK = new Param[Int](this, "topK",
```

Comment:
you need to use IntParam instead, see other params

Reply:
I've fixed it.